jobbergate-agent/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "jobbergate-agent"
-version = "5.0.0a1"
+version = "4.7.0"
```

Comment:
That is not expected to change in a feature branch, what happened here?

Reply:
Oh, hell, that was leftover from QA'ing the auto-update stuff. Will restore.

Reply:
Fixed the error, amended the commit, and force-pushed.